package.json

```diff
@@ -1,6 +1,6 @@
 {
```

Comment:
Please revert all changes to this file.

Package name should not change, and the version will be updated via the release process.

The two extra dependencies added are subpackages of this one, and are already added in the appropriate `package.json`.